api/scripts/run-docker.sh

```diff
@@ -59,6 +62,9 @@ if [ "$1" == "migrate" ]; then
 elif [ "$1" == "serve" ]; then
     if [ $# -eq 2 ]; then go_to_sleep "$2"; fi
     serve
+elif [ "$1" == "run-tasks" ]; then
```

Comment:
I'd say that run-processor or run-task-processor would be better here. run-tasks sounds like an atomic operation rather than a long running process.

Reply:
Agreed. Fixed.